gateway-js/CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ Since the gateway itself is no longer responsible for composition:
 
 Some defensive code around gateway shutdown has been removed which was only relevant to users who are running the gateway within `ApolloServer` before v2.18. If you are still running one of these versions, server shutdown may not happen as smoothly.
 
+Upgrading dependency to graphql16
```

Comment:
The changelog above is pretty atypical, much longer than most. Check out the format for typical entries further down.

This should be moved below the PR link on L26 and include its own PR link after the description